controllers/ovnnorthd_controller.go

```diff
@@ -439,7 +399,7 @@ func (r *OVNNorthdReconciler) reconcileNormal(ctx context.Context, instance *ovn
 
 	// Define a new Deployment object
 	depl := deployment.NewDeployment(
-		ovnnorthd.Deployment(instance, serviceLabels, serviceAnnotations, nbEndpoint, sbEndpoint, envVars),
+		ovnnorthd.Deployment(instance, serviceLabels, map[string]string{}, nbEndpoint, sbEndpoint, envVars),
```

Comment:
this argument is no longer needed, so can be cleaned up I think.

Reply:
This is not addressed.

Reply:
Ah sorry! Missed this